operator/README.md

```diff
@@ -0,0 +1,120 @@
+# DNSNameResolver Operator
+The DNSNameResolver Operator can be used to test the unmanaged DNSNameResolver conroller.
+
+## Description
+The DNSNameResolver controller sends DNS name lookup requests to maximum 5 random CoreDNS
+pods whenever the TTL of the IP addresses corresponding to a DNS name expires. The DNS name
+lookups will be handled by the ocp-dnsnameresolver CoreDNS plugin and any new IP address
+corresponding to the DNS name will be added to the status of the DNSNameResolver CR. The
+controller also removes the stale IP addresses from the status of the DNSNameResolver CRs
+after a grace period.
+
+To test the unmanaged DNSNameResolver controller, the operator runs another controller which
+watches the DNSNameResolver CRD and starts the DNSNameResolver controller.
+
+Kindly provide correct values for the following arguments when deploying the operator:
```

Comment:
Where the user should put those values?

Reply:
The user should provide the arguments through the `args` field of the manager deployment.

Reply:
We should mention this in the README, just for the clarity of the instructions.